get_id.js

```diff
@@ -64,175 +76,88 @@ async function getEP_write_placeholders (id) {
 }
 
 
-function put_placeholder(name, placeholder) {
+function put_placeholder_values(name, placeholder) {
     let selector = $(`input[name=${name}]:visible`)
+    console.log(name)
+    console.log(placeholder)
     if (selector[0] !== undefined) {
         if (selector[0].placeholder !== undefined) {
             console.log(placeholder)
             selector[0].placeholder = placeholder
-        } else {
-            selector[0].value = placeholder
-            console.log("select " + placeholder)
         }
     } else {
-        console.log(name + "undefined")
-        $(`[name=${name}]`).val(placeholder)
+        console.log(name + " placeholder undefined")
+        // $(`[name=${name}]`).val(placeholder)
+    }
+}
+
+function put_value_values(name, placeholder) {
+    let selector = $(`select[name=${name}]`).val();
+    if (selector !== undefined) {
+        if (selector[0].val() !== undefined) {
+            console.log(placeholder)
+            selector[0].val(placeholder)
+        }
+    } else {
+        console.log(name + " select undefined")
+        // $(`[name=${name}]`).val(placeholder)
     }
 }
 
 function parse_disperse_filler (resp) {
-    let filler_dispers = ["marka", "manufacturer", "chemical_nature", "chemical_nature_iso", "purity_of_disperse_filler", "purity_of_disperse_filler_iso", "dispersity", "dispersity_iso", "moisture_content", "moisture_content_iso", "density", "density_iso"]
-    filler_dispers.forEach(function(item, i) {
-        // let placeholder = resp[item]
-        // console.log(item)
-        // console.log(placeholder)
-        put_placeholder(item, resp[item])
-    });
+    let massivofplaceholders = ["marka", "manufacturer", "chemical_nature", "chemical_nature_iso", "purity_of_disperse_filler", "purity_of_disperse_filler_iso", "dispersity", "dispersity_iso", "moisture_content", "moisture_content_iso", "density", "density_iso"]
+    massivofvalue = undefined
+    function_foreach (massivofplaceholders, massivofvalue, resp)

```

Comment:
make resp as first parameters than last parameters may to ignore
function_foreach (resp, massivofplaceholders)